기능추가 : 보드 출력시 콘솔 초기화

shwon79 opened the pull request:
### 간결한 설명
- 보드판을 출력 할 시 콘솔창을 초기화해 가장 최신의 보드판만 콘솔창에 표시하도록 한다.

### 구체적인 설명
- 출력 시에 콘솔 초기화를 위해 HumanTurn함수 마지막에 system("cls")를 추가 했음.
- 콘솔창은 vs컴퓨터라면 플레이어가 차례를 마친 후 최신의 보드판을 출력하고 아래에 컴퓨터의 차례가 끝난 후의 보드판을 보여준다.
- vs사람이라면 가장 최신의 보드판 한개만 콘솔창에서 보여준다.

### 스크린샷
변경 전
![초기화 변경 전](https://user-images.githubusercontent.com/55779075/69477595-7b1b7100-0e2b-11ea-9367-c8aaf2dc1a2a.PNG)

변경 후
![초기화 변경 후](https://user-images.githubusercontent.com/55779075/69477598-8078bb80-0e2b-11ea-8f7b-cf6c2fe52408.PNG)


###  *추가적인 사항*

shwon79 committed:
기능추가 : 사람 vs 사람

변경 전
 - 틱택토 게임을 사람 vs 컴퓨터만 플레이 가능함.
변경 후
 - 기존 틱택토 게임에 사람 vs 사람 모드를 넣어 사용자가 선택하여 사람 vs
 컴퓨터 또는 사람 vs 사람을 할 수 있음.

shwon79 committed:
기타 : 커밋 메시지 등록 #13

변경 전
 - 사람 vs 사람 기능을 머지하는 과정에서 커밋 메시지가 작성되지 않았음.
변경 후
 - 커밋 메시지를 수정해 어떤 기능을 머지했는지 표시함.

관련 이슈
 - #13

shwon79 committed:
기타 : 로컬저장소 Fetch후 Merge

변경 전
 - dev 브랜치가 최신의 dev 브랜치로 업데이트가 되어있지 않음.
변경 후
 - fetch명령어로 원격저장소의 커밋을 받아온 후 로컬의 dev브랜치에
 merge하여 dev브랜치를 갱신함.

shwon79 committed:
기능추가 : 보드판 출력시 콘솔 초기화

변경 전
 - 보드를 출력할 때 보드를 콘솔창에 이어서 출력하기 때문에 콘솔창이
 지저분하고 한 눈에 보기 어려웠음.
변경 후
 - 보드를 출력할 때 콘솔창을 초기화하여 콘솔 창에는 컴퓨터가 차례를
 끝냈을 때의 보드와 플레이어가 차례를 마쳤을 때의 보드만 존재함.
세부사항
 - 콘솔창을 비우기 위해 system("cls")를 사용하였음.

GitKosy commented:
컴퓨터 턴이 종료될때도 보드가 초기화되는식으로 되면 좋을거같아.

hkmast commented:
@GitKosy 컴퓨터 턴 종료될때 보드가 초기화 되면 보드 자체가 안보이는거 아니야?
@shwon79 이거 만약에 사람대 사람 하거나 사람이 후공이면 이상해질것 같은데

hkmast commented:
다시 확인해 봤는데 난 괜찮은것 같음

hkmast commented:
> 다시 확인해 봤는데 난 괜찮은것 같음

shwon79 committed:
기능수정 : 보드 출력시 콘솔 초기화

변경 전
 - 보드판을 플레이어  차례가 끝난 후의 보드판과 컴퓨터의 차례가 끝난 후의
 보드판 2개를 콘솔 창에서 보여주었다.
변경 후
 - 컴퓨터의 차례가 끝난 후의 보드판 1개만 콘솔 창에서 볼 수 있다.
